src/System.Private.CoreLib/shared/System/Globalization/CultureData.Windows.cs

```diff
@@ -428,13 +425,14 @@ internal static string ReescapeWin32String(string str)
             return result.ToString();
         }
 
-        internal static string[] ReescapeWin32Strings(string[] array)
+        internal static string[]? ReescapeWin32Strings(string[]? array)
         {
             if (array != null)
             {
                 for (int i = 0; i < array.Length; i++)
                 {
-                    array[i] = ReescapeWin32String(array[i]);
+                    // only returns null when null is passed
```

Comment:
So should the type of the array be `string?[]?` if it both might be null and contain nulls?

Reply:
IMO this line does not influence the nullability:
- array[i] is maybe null - we try to find out, based on code except for this line there is no reason to claim it's nullable
- ReescapeWin32String returns null only if null is passed

given that I think this should stay non-nullable